docling/models/easyocr_model.py

```
@@ -85,7 +85,8 @@ def __init__(
     @staticmethod
     def download_models(
         detection_models: List[str] = ["craft"],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)